Model/src/GamePackage/Bank.java

```diff
@@ -1,7 +1,158 @@
+    public int getNumHouses() {
+        return numHouses;
+    }
+
+    /****************************************************************************************************************
+     * Sets the numHouses
+     *
+     * @param numHouses
+     ***************************************************************************************************************/
+    public void setNumHouses(int numHouses) {
+        this.numHouses = numHouses;
+    }
+
+    /****************************************************************************************************************
+     * Returns the number of hotels.
+     *
+     * @return
+     ***************************************************************************************************************/
+    public int getNumHotels() {
+        return numHotels;
+    }
+
+    /****************************************************************************************************************
+     * Sets the number of hotels.
+     *
+     * @param numHotels
+     ***************************************************************************************************************/
+    public void setNumHotels(int numHotels) {
+        this.numHotels = numHotels;
+    }
+
+    /****************************************************************************************************************
+     * Returns the  list of properties owned.
+     *
+     * @return
+     ***************************************************************************************************************/
+    public ArrayList<PropertySquare> getPropertiesOwned() {
+        return propertiesOwned;
+    }
+
+    /****************************************************************************************************************
+     * Sets the current properties owned.
+     *
+     * @param propertiesOwned
+     ***************************************************************************************************************/
+    public void setPropertiesOwned(ArrayList<PropertySquare> propertiesOwned) {
+        this.propertiesOwned = propertiesOwned;
+    }
+
+    /****************************************************************************************************************
+     * Returns the list of mortgaged properties
+     *
+     * @return
+     ***************************************************************************************************************/
+    public ArrayList<PropertySquare> getPropertiesMortgaged() {
+        return propertiesMortgaged;
+    }
+
+    /****************************************************************************************************************
+     * Set the list of properties owned
+     *
+     * @param propertiesMortgaged
+     ***************************************************************************************************************/
+    public void setPropertiesMortgaged(ArrayList<PropertySquare> propertiesMortgaged) {
+        this.propertiesMortgaged = propertiesMortgaged;
+    }

```

Comment:
This ArrayList type also needs to be changed

Reply:
other than Ownable properties, what else can be mortgaged ?

Reply:
nothing. But as it is, you have the type of PropertySquare. It will need to be OwnableSquare when we build that class

Reply:
add this to an issue for the OwnableSquare.